build/charts/antrea/conf/antrea-controller.conf

```diff
@@ -71,3 +71,25 @@ nodeIPAM:
   # or when IPv6 Pod CIDR is not configured. Valid range is 64 to 126.
   nodeCIDRMaskSizeIPv6: {{ .nodeCIDRMaskSizeIPv6 }}
 {{- end }}
+
+ipsecSigner:
+{{- with .Values.ipsecSigner }}
+  # Enable the signer controller within the Antrea controller.
+  enable: {{ .enable }}
+  # Determines the auto approval policy of Antrea signer for IPsec certificates management.
+  # It has the following options:
+  # never:                          Controller will never auto-approve CertificateSingingRequests and they need
+  #                                 to be approved manually by `kubectl certificate approve`
+  # validateAgentPodName (default): Controller will auto-approve the CertificateSingingRequest if the creator can
+  #                                 be validated. This ensures that each Agent can only request certificates for
+  #                                 its own Node. This requires `BoundServiceAccountTokenVolume` feature to
+  #                                 be enabled.
+  # always:                         Controller will auto-approve the CertificateSingingRequest without checking
+  #                                 the identity of the creator.
+  autoApprovePolicy: {{ .autoApprovePolicy | quote }}
+  # Indicates whether to use auto-generated self-signed CA certificate.
+  # If false, a Secret named "antrea-ipsec-ca" must be provided with the following keys:
+  #   ca.crt: <CA certificate>
+  #   ca.key: <CA private key>
```

Comment:
There is a type of secret: "kubernetes.io/tls", it expects `tls.crt` and `tls.key` field.
`kubectl create secret tls` can create such a secret. do you think it's more suitable?

Reply:
I am not quite confident to use Secret of `kubernetes.io/tls` type as`tls.crt` and `tls.key` are mandatory fields. I remember such Secret may also contain `ca.crt` fields for CA certificates. Do you think it is ok to use `tls.key` and `tls.crt` for root private key and certificate?

Reply:
Does "tls" mean non CA certificate specifically? If not, tls.crt and tls.key can just map to ca cert and ca key? @jianjuns do you have an idea?

Reply:
I think we can use TLS Secret.

Reply:
Thanks. updated.